### ISSUE TITLE
I am new, where do I start? 

### ISSUE THREAD

**Issue by rshade** [sous-chefs/meta#84]
are you trying to solve a specific problem?
are you already using any of the tools or cookbooks in sous-chefs?

**Comment by xamebax** [comment]
Expressing my interest.

**Comment by martinb3** [comment]
Also been here a while (since inception), but haven't had a chance to really jump in & contribute. But I'd love to.

**Comment by josephholsten** [comment]
I'm not new, but I want to make sure new folks find something they can do. If you're new, it would be good for me to know:
- are you trying to solve a specific problem?
- are you already using any of the tools or cookbooks in sous-chefs?

**Comment by damacus** [comment]
I'd also like to know if there's something *not* in Sous-Chefs that you think could do with a community of people behind it. i.e. is there an abandoned cookbook you'd like adopting?

**Comment by rljohnsn** [comment]
What is a good candidate for sous-chefs cookbook? Can it be one a group has not open sourced yet but has wanted too? i.e. Is this a good place to introduce a new cookbook for the community?

**Comment by damacus** [comment]
Yeah. I'd encourage any user to get started here. We're a bunch of people who've collectively open sourced a lot of stuff. 

The best thing about it, is if you get to a point where you want o go your own way, that's also possible.

**Comment by aried3r** [comment]
I'm currently trying to make the [ruby_rbenv](https://github.com/sous-chefs/ruby_rbenv) cookbook work with Chef 13.

I followed [the contributing guide](http://sous-chefs.org/contributing/) but quickly got stuck trying to run `kitchen test` as I was at that point unaware that ChefDK comes with its own ruby and was conflicting with my rbenv install.

Since I'm pretty new to contributing to chef cookbooks (only been a user of Chef until now except for very tiny contributions here and there) I'd have loved a link to ChefDK and maybe a little explanation. @cheeseplus was quick to point out that for me prepending `chef exec` will work for me.

It's not that different from the workflow I'm used to from Rails and other ruby gems, but seeing a `Gemfile` I quickly assumed the workflow is the same, using `bundle install` etc.

TL;DR: Should have googled more and RTFM, but a more exhaustive contributing guide would have helped me a great deal. :)

**Comment by damacus** [comment]
A generic getting started guide in the test kitchen documentation might
help here,and link to it.

(and if its there already link to it)

**Comment by mateusduboli** [comment]
I've became a maintainer of a cookbook here in sous-chefs, how do I publish the changes in the supermarket?

**Comment by josephholsten** [comment]
@mateusduboli that is so important! I created #89 for that particular issue, but we really need someone to draft the easiest process that could possibly work.

**Comment by damacus** [comment]
@mateusduboli pester me! #sous-chefs on slack is usually the best place as the notifications work better on there. And I can pickup work a sync. I have trouble keeping up with github! (also I don't monitor here as often)

**Comment by josephholsten** [comment]
if you've got any newb-friendly tickets, please put them onto https://github.com/orgs/sous-chefs/projects/5!

**Comment by damacus** [comment]
Do we want this ticket open still? 🤔

**Comment by josephholsten** [comment]
@damacus What's the permanent answer to this question? where is it documented? I'll post & pin it in the chat room

**Comment by damacus** [comment]
Nope, it doesn't look like there is...
